IPython/html/static/notebook/js/kernelselector.js

```diff
@@ -78,7 +78,7 @@ define([
             new_notebook_submenu.append(
                 $("<li>").attr("id", "new-notebook-submenu-"+ks.name).append(
                     $('<a>')
-                        .attr('href', '#')
+                        .attr('href', 'javascript:void(0)')
```

Comment:
There are a couple of justifications here. Since I'm trying to listen for events that have a fragment identifier, links with `href="#"` are picked up by the listener. One possibility would be to restrict the elements in which the listener is working on. The second justification is that semantically doesn't make sense to have `href="#"` for a link. These buttons are not placeholders nor are they going to be crawled by search engines. As I said before, my preference is to use button elements, but I think that is going to break a lot of things.